mimalloc-sys-test/build.rs

```diff
@@ -2,14 +2,13 @@
 
 extern crate ctest;
 
-use std::{env, path::PathBuf};
-
 fn main() {
-    let root = PathBuf::from(env::var_os("DEP_MIMALLOC_ROOT").unwrap());
-    eprintln!("ROOT={:?}", root);
     let mut cfg = ctest::TestGenerator::new();
     cfg.header("mimalloc.h")
-        .include(root.join("lib").join("mimalloc-1.0").join("include"))
+        .include(concat!(
+            env!("CARGO_MANIFEST_DIR"),
+            "/../mimalloc-sys/mimalloc/include"
```

Comment:
This is incorrect. The `mimalloc.h` that must be used needs to be the one that's installed.

Reply:
I'll see I can get DEP_MIMALLOC_ROOT working.  I got this style from flate2-rs's [systest/build.rs](https://github.com/alexcrichton/flate2-rs/blob/master/systest/build.rs).

Reply:
Installing the library can modify the headers, e.g., the library could easily allow generating a header that uses `malloc` instead of `mi_malloc` (or `rust_malloc` like jemalloc does) in the future. Using the headers from the installed location avoids any problems.